scripts/windows_install_mingw64.ps1

```diff
@@ -19,9 +19,17 @@ msys 'pacman --noconfirm -Scc' ;
 
 echo 'installing packages' ;
 msys 'pacman -S --needed --noconfirm git bison flex make diffutils \
-    ucrt64/mingw-w64-ucrt-x86_64-{ccache,gcc,icu,libxml2,libxslt,lz4,meson,perl,pkg-config,zlib}' ;
+    ucrt64/mingw-w64-ucrt-x86_64-{ccache,gcc,icu,libxml2,libxslt,lz4,make,meson,perl,pkg-config,zlib}' ;
 msys 'pacman -Scc --noconfirm'
 
-# Install perl modules to enable tap tests
+### Install perl modules to enable tap tests
 msys 'where perl'
-msys '(echo; echo o conf recommends_policy 0; echo notest install IPC::Run) | cpan'
+
+echo 'Install cpanm'
+# Autoconfigure first
+msys 'echo | cpan'
+msys 'cpan App::cpanminus'
```

Comment:
Well, now this can fail to install without erroring out :)

I'd probably just leave it as before, but just add a `perl -mIPC::Run -e 1` as a separate command - that'll fail if IPC::Run isn't installed.